src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -105,7 +107,10 @@ trait Backend {
     def outputAutoConversion(callFqn: String, taskOutput: TaskOutput, rawOutputValue: WdlValue): Try[WdlValue] = {
       rawOutputValue match {
         // Autoconvert String -> File.
-        case v: WdlString if taskOutput.wdlType == WdlFileType => Success(WdlFile(hostAbsoluteFilePath(v.value)))
+        case v: WdlString if taskOutput.wdlType == WdlFileType =>
+          val path = hostAbsoluteFilePath(v.value)
+          if (Files.exists(Paths.get(path))) Success(WdlFile(path))
+          else Failure(new RuntimeException(s"Error attempting to process outputs of call $callFqn:\nCannot process output '${taskOutput.name}' because of error:\nInvalid path: $path"))
```

Comment:
Super long line!  Something like "expected output file '$yourfile' not found" might be more informative.